changelog/fragments/add-plugins.yaml

```
@@ -0,0 +1,7 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Add `kustomize/v1` plugin which scaffolds the init common base. More info: TBD
```

It's ok to omit the `More info` bit for now
```suggestion
      Added `kustomize.common/v1` plugin which scaffolds the a commonly used project base that leverages `kustomize`.
```

---

done